src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -558,22 +673,194 @@ def parse_chat(chat_str, images: List = None, valid_roles: List[str] = None):
                     )
                 # "name" is optional for other role types.
                 else:
-                    last_message["content"] = to_content_str_or_list(chunk, hash2images)
+                    last_message["content"] = to_content_str_or_list(chunk, hash2images, image_detail)
             else:
                 last_message["name"] = parsed_result[0]
-                last_message["content"] = to_content_str_or_list(parsed_result[1], hash2images)
+                last_message["content"] = to_content_str_or_list(parsed_result[1], hash2images, image_detail)
```

Comment:
This is new feature to support image detail for gpt vision tool. Please modify PR title and description

Reply:
Prompty doesn't support image. In order to ensure that the logic of prompty is similar to that in common, the image parameter in common is retained. After the build, prompty will support image.